src/stories/Library/event-list-item/event-list-item.scss

```diff
@@ -0,0 +1,176 @@
+.event-list-item {
+  @include show-svg-on-hover();
+  text-decoration: none;
+  display: grid;
+  padding: $s-lg;
+  background-color: #fcfcfc;
+  margin: 0 auto;
+  grid-template-areas:
+    "image"
+    "content";
+  grid-template-rows: auto;
+  row-gap: $s-md;
+  max-width: $layout__max-width;
+
+  > svg {
+    display: none;
```

Comment:
Consider if this is necessary.

The `show-svg-on-hover` mixin already has ` > svg { visibility: hidden }` by default. If that is not enough here then please add a comment explaining why.

If this is a general problem then I think we should address it if the mixin if possible.

Reply:
The element would still be rendered in mobile even tho the visibility was hidden, and i didn't want that, but I see that the code for ```show-svg-on-hover``  has been changed. 

I did another ticket, before the refactor of arrows, and now i just did it the same way. 

but i can see that benja has added
``` @include media-query__small("max-width") {
      display: none;
    }```
    to the
     
    ```show-svg-on-hover``` mixin. 
    
    which makes my code double confetti. 
    
    it has been removed! 🥳